### PR TITLE
[TACHYON-748] Update Javadoc in TachyonFSTestUtils

### PR DESCRIPTION
**Pull request by chen-tian**
https://tachyon.atlassian.net/browse/TACHYON-748

### CODE REVIEW

**Comment by apc999** [comment]
Hi @chen-tian  thanks for the patch!
Could you put JIRA number in the title (e.g., [TACHYON-748] Update Javadoc in TacjyonFSTestUtils) and also put the link to this JIRA in  description?

**Comment by apc999** [comment]
Jenkins, test it please

**Comment by gpang** [comment]
ok to test

File: clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java [hunk 1]
```diff
@@ -87,6 +87,15 @@ public static int createByteFile(TachyonFS tfs, String fileName, WriteType op, i
     return fileId;
   }
 
+  /**
+   * List files at give <code>path</code>.
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java]: "give" -> "given"

File: clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java [hunk 2]
```diff
@@ -87,6 +87,15 @@ public static int createByteFile(TachyonFS tfs, String fileName, WriteType op, i
...
+   *
+   * @param tfs a TachyonFS handler
+   * @param path a path in tachyon file system 
+   * @return list of files
+   * @throws IOException
+   */
+
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java]: Could you remove this extra blank line?

**Comment by gpang** [comment]
Thanks for the fixes! I added a few minor comments.

File: clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java [hunk 3]
```diff
@@ -26,11 +26,11 @@
...
-   * @param fileName
-   * @param op
-   * @param len
-   * @return created file id.
+   * @param tfs a TachyonFS handler
+   * @param fileName the name of the file to be created
+   * @param op WriteType used to create the file 
+   * @param len file size
+   * @return Id of the created file
    * @throws IOException
```
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java]: Could you please also add the conditions when the exceptions are thrown?

File: clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java [hunk 4]
```diff
@@ -87,6 +87,14 @@ public static int createByteFile(TachyonFS tfs, String fileName, WriteType op, i
     return fileId;
   }
 
+  /**
+   * List files at a given <code>path</code>.
+   *
+   * @param tfs a TachyonFS handler
+   * @param path a path in tachyon file system 
+   * @return list of files
```
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java]: Perhaps `a list of Strings representing the file names under the given path`?

**Comment by uronce-cc** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**